ts/components/conversation/media-gallery/DocumentListItem.tsx

```diff
@@ -7,6 +7,7 @@ interface Props {
   fileName?: string | null;
   fileSize?: number;
   i18n: (key: string, values?: Array<string>) => string;
+  isLast: boolean;
```

Comment:
Could also do this with CSS... Just sayin' :0)

Reply:
With selectors, yes, but what about CSS-in-JS? Is there an easy way?